nixos/modules/flyingcircus/packages/fcmaintenance/setup.py

```diff
@@ -52,6 +52,7 @@ def run_tests(self):
     cmdclass={'test': PyTest},
     extras_require={
         'dev': test_req + ['pytest-cov'],
```

Comment:
interesting. cannot remember. Keep it simple. Maybe something implicitly installed [test]? But really, cannot remember.